t/ui/03-setting-links.t

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env perl
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+use Test::Most;
+
+use FindBin;
+use lib "$FindBin::Bin/../lib";
+use Test::Mojo;
+use Test::Warnings ':report_warnings';
+use OpenQA::Test::Case;
+use Mojo::File qw(path tempfile);
+
+my $schema = OpenQA::Test::Case->new->init_data;
+# setup openqa.ini with job_settings_ui
+local $ENV{OPENQA_CONFIG} = "t/data/03-setting-links";
```

Comment:
Do you really need `local` here?

Reply:
no really. i thought that it will make the scope clearer. i will remove it